installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -55,6 +55,9 @@ include::modules/installing-ocp-agent-ibm-z-kvm.adoc[leveloffset=+2]
 // Adding {ibm-z-title} Logical Partition (LPAR) as agents
 include::modules/adding-ibm-z-lpar-agent.adoc[leveloffset=+2]
 
+// Adding {ibm-z-title} agents in a Logical Partition (LPAR) as a day 2 operation
+include::modules/adding-ibm-z-lpar-agent-day-2.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
 .Additional resources
```

Comment:
Keep in mind that this is a module-level additional resources section, not an assembly-level one. So this list of links might apply to the section [right above](https://88076--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.html#adding-ibm-z-lpar-agents_prepare-pxe-assets-agent) your new module. If that is the case, it would probably make more sense to add your module below this additional resources section.

Reply:
Applies to both but you made me aware that the link leads to z/VM instead of LPAR.

Reply:
Rethinking this I need to correct myself the additional resources link applies to day 1 will move 👍